src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1015,7 +1015,7 @@
     {
       "run": "after",
       "snippetPath": "statisticalCodeTypesReferenceCheckTrigger.sql",
-      "fromModuleVersion": "23.0.0"
+      "fromModuleVersion": "24.0.4"
```

Comment:
The next targeted release on the 24.0.x branch is 24.0.3. Why was 24.0.4 chosen for the version where the script should run?

Reply:
updated to 24.0.3